.github/workflows/tests.yml

```diff
@@ -393,15 +393,28 @@ jobs:
     - name: Install SCT
       # NB: In real-world usage, it's assumed that the user would download the
       # install_sct.bat file from the Downloads page, and _not_ the full repo.
-      # Then, the repo would be cloned to the working directory.
-      # However, this installation method doesn't allow us to test changes
-      # from forks. So, we use an in-place install instead.
-      run: install_sct.bat
+      # Then, the repo would be cloned to a tmpdir, then copied to the destination folder.
+      # However, this only really works on `master`, since for PRs, there is a chance
+      # that the PR is coming from a fork. And, in that case, we won't be able to clone
+      # the proper branch, as it exists only on the fork.
+      run: |
+        if %GITHUB_REF_NAME%=="master" (
+          if not exist %HOME%\Downloads mkdir %HOME%\Downloads
```

Comment:
Good question! I started out with `%USERPROFILE%` (which refers to `C:\` on GHA runners). But, then when I tried to `cd` into `%USERPROFILE%`, it just... wouldn't? No log message, no error, just... staying in the current directory.

There are a lot of force pushes on this PR, and many of them were trying to debug why I wasn't able to `cd` into the `C:\`-based path, but eventually I tried `%HOME%` (which I believe refers to a `D:\` based path on GHA runners) and it worked, so I thought I'd cut my losses instead of debugging further.

(Since it's just a made-up folder, and the script location can be anywhere as long as it triggers the `git clone`, I figured `%HOME%` would be sufficient.)

Reply:
Oh. I may be a bit silly here. I think this is an intentional part of Windows: https://superuser.com/questions/135214/using-cd-command-in-windows-command-line-cant-navigate-to-d

I might try doing this instead:

```
C:
cd %USERPROFILE%\Downloads
```

Reply:
Yep! `/d` worked. 

((Even though the test was a failure, that's only because I forgot to uncomment out the `master` conditional for the path echoing step. During the actual installation, the "standalone install" is [triggered correctly](https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/5094870665/jobs/9159200221#step:3:24), even with `%USERPROFILE%`.))

Reply:
Oh yeah, `cd /d`! It's so easy to forget about it.